symbology/graphics-overlay-dictionary-renderer-3D/README.md

```diff
@@ -1,6 +1,6 @@
 # Graphics overlay dictionary renderer 3D
```

Comment:
Maybe more of a samples team / renaming discussion, but interesting that we have "Dictionary renderer graphics overlay" and then "graphics overlay dictionary renderer 3D" (and not "Dictionary renderer graphics overlay 3D") and they're in different packages.

Reply:
totally agree. there is a discrepancy between the different SDKs in that some have the 2D sample, the 3D sample or both. this will be revisited in the future and one sample will probably be removed in favour of another, so for now we'll keep things as they are.